os/kernel/task/task_terminate.c

```diff
@@ -171,8 +171,6 @@ int task_terminate(pid_t pid, bool nonblocking)
 		return -ESRCH;
 	}
 
-	trace_begin(TTRACE_TAG_TASK, "task_terminate");
```

Comment:
Sorry but need to remove entire ttrace code. not only here.

Reply:
Let's revert this pr and do with new pr for this.

Reply:
Oh okay. I understood it wrong. I dropped this commit.